arm-recoveryservicesbackup/2016-06-01/swagger/recoveryservicesbackup.json

```diff
@@ -0,0 +1,1548 @@
+          "type": "string"
+        }
+      }
+    },
+    "Resource[Job]": {
+      "description": "Resource wrapper for \"T\" type",
+      "type": "object",
+      "properties": {
+        "id": {
+          "description": "Resource Id represents the complete path to the resource",
+          "type": "string"
+        },
+        "name": {
+          "description": "Resource name associated with the resource",
+          "type": "string"
+        },
+        "type": {
+          "description": "Resource type represents the complete path of the form Namespace/ResourceType/ResourceType/...",
+          "type": "string"
+        },
+        "location": {
+          "description": "Resource location",
+          "type": "string"
+        },
+        "tags": {
+          "description": "Resource tags",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "eTag": {
+          "description": "Optional ETag",
+          "type": "string"
+        },
+        "properties": {
+          "$ref": "#/definitions/Job",
+          "description": "Resource properties"
+        }
+      }
+    },
+    "Job": {
+      "description": "Defines workload agnostic properties for a job.",
+      "type": "object",
+      "properties": {
+        "entityFriendlyName": {
+          "description": "Friendly name of the entity on which the current job is executing.",
+          "type": "string"
+        },
+        "backupManagementType": {
+          "description": "Backup management type to execute the current job. Gets or sets the BackupManagementType to IaasVM, DPM or MAB",
+          "type": "string"
+        },
+        "operation": {
+          "description": "Gets or sets the operation name.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Job status",
+          "type": "string"
+        },
+        "startTime": {
+          "format": "date-time",
+          "description": "Gets or sets the start time.",
+          "type": "string"
+        },
+        "endTime": {
+          "format": "date-time",
+          "description": "Gets or sets the end time.",
+          "type": "string"
+        },
+        "activityId": {
+          "description": "ActivityId of job",
+          "type": "string"
+        },
+        "jobType": {
+          "description": "This property will be used as the discriminator for deciding the specific types in the polymorhpic chain of types",
+          "type": "string"
+        }
+      },
+      "discriminator": "jobType"
+    },
+    "JobQueryObject": {
+      "description": "Filters to list the jobs.",
+      "type": "object",
+      "properties": {
+        "status": {
+          "description": "Status of the job. \r\n            Possible values: InProgress, Completed, Failed, CompletedWithWarnings, Cancelling, Cancelled.",
```

Comment:
Please remove \r and \n from the description. AutoRest will automatically format it.